sessionctx/variable/session.go

```diff
@@ -1534,6 +1534,12 @@ type SessionVars struct {
 	// When set to true, skip missing partition stats and continue to merge other partition stats to global stats.
 	// When set to false, give up merging partition stats to global stats.
 	SkipMissingPartitionStats bool
+
+	// OptObjective indicates whether the optimizer should be more stable, predictable or more aggressive.
+	// For now, the possible values and corresponding behaviors are:
+	// OptObjectiveModerate: The default value. The optimizer considers the real-time stats (real-time row count, modify count).
+	// OptObjectiveDetermined: The optimizer doesn't consider the real-time stats.
+	OptObjective string
```

Comment:
The var name seems that it will deal with other things besides the statistics?

Reply:
It's decided by the PM. He's planning broader usage for this variable.